media-sound/forked-daapd/forked-daapd-23.2.ebuild

```diff
@@ -49,6 +49,9 @@ pkg_setup() {
 }
 
 src_prepare() {
+	# Required until upstream accepts https://github.com/ejurgensen/forked-daapd/pull/179
```

Comment:
Comment isn't necessary as I usually drop all patches on a bump (my style).  Glad to see you pushed the patch upstream as well as dropping it on the ebuild.